spec/bundle/unpatched_gems/Gemfile.lock

```diff
@@ -17,7 +17,14 @@ GEM
     concurrent-ruby (1.1.7)
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
+    mini_portile2 (2.8.0)
     multi_json (1.15.0)
+    nokogiri (1.13.6)
+      mini_portile2 (~> 2.8.0)
+      racc (~> 1.4)
+    nokogiri (1.13.6-x86_64-linux)
+      racc (~> 1.4)
```

Comment:
If we are going to test `Gemfile.lock` containing gems from multiple platforms or gems which explicit platforms set in the `Gemfile`, I think that should be a separate `spec/bundle/` directory (ex: `spec/bundle/unpatched_multi_platform/`).